### PR TITLE
Version Packages (preview)

### PR DESCRIPTION
**Pull request by frontsidejack**
This PR was opened by the [Changesets release](https://github.com/changesets/action) GitHub action. When you're ready to do a release, you can merge this and publish to npm yourself or [setup this action to publish automatically](https://github.com/changesets/action#with-publishing). If you're not ready to do a release yet, that's fine, whenever you add more changesets to v2, this PR will be updated.

⚠️⚠️⚠️⚠️⚠️⚠️

`v2` is currently in **pre mode** so this branch has prereleases rather than normal releases. If you want to exit prereleases, run `changeset pre exit` on `v2`.

⚠️⚠️⚠️⚠️⚠️⚠️

# Releases
### Minor Changes

-   9cf6053: Increase default value of max subscribers on channel
-   0b24415: Add WritableStream interface and implement it for channels

### Patch Changes

-   Updated dependencies [0b24415]
-   Updated dependencies [22e5230]
-   Updated dependencies [70c358f]
-   Updated dependencies [3983202]
-   Updated dependencies [2c2749d]
    -   @effection/subscription@2.0.0-preview.5
    -   @effection/core@2.0.0-preview.5

 ## @effection/core@2.0.0-preview.5

### Minor Changes

-   70c358f: Store root in a global variable

 ## effection@2.0.0-preview.6

### Minor Changes

-   9cf6053: Increase default value of max subscribers on channel
-   0b24415: Add WritableStream interface and implement it for channels
-   442f220: New API for exec and daemon which does not take scope directly
-   22e5230: Add `join` method on stream to return stream result
-   70c358f: Store root in a global variable
-   3983202: Add `stringBuffer` method to stream which buffers stream to a string
-   2c2749d: Add a `buffer` method on Stream to buffer stream contents for later replay

### Patch Changes

-   Updated dependencies [9cf6053]
-   Updated dependencies [0b24415]
-   Updated dependencies [22e5230]
-   Updated dependencies [70c358f]
-   Updated dependencies [3983202]
-   Updated dependencies [2c2749d]
    -   @effection/channel@2.0.0-preview.5
    -   @effection/subscription@2.0.0-preview.5
    -   @effection/core@2.0.0-preview.5

 ## @effection/mocha@2.0.0-preview.4

### Minor Changes

-   395cb3e: Add it.only and it.skip

### Patch Changes

-   Updated dependencies [70c358f]
    -   @effection/core@2.0.0-preview.5

 ## @effection/node@2.0.0-preview.6

### Minor Changes

-   442f220: New API for exec and daemon which does not take scope directly

### Patch Changes

-   Updated dependencies [9cf6053]
-   Updated dependencies [0b24415]
-   Updated dependencies [22e5230]
-   Updated dependencies [70c358f]
-   Updated dependencies [3983202]
-   Updated dependencies [2c2749d]
    -   @effection/channel@2.0.0-preview.5
    -   @effection/subscription@2.0.0-preview.5
    -   @effection/core@2.0.0-preview.5

 ## @effection/subscription@2.0.0-preview.5

### Minor Changes

-   0b24415: Add WritableStream interface and implement it for channels
-   22e5230: Add `join` method on stream to return stream result
-   3983202: Add `stringBuffer` method to stream which buffers stream to a string
-   2c2749d: Add a `buffer` method on Stream to buffer stream contents for later replay

### Patch Changes

-   Updated dependencies [70c358f]
    -   @effection/core@2.0.0-preview.5

### CODE REVIEW

**Review by minkimcello — CHANGES_REQUESTED** [review]
We need to merge #267 first.

**Comment by cowboyd** [comment]
@minkimcello #267 is merged. Does that mean this is a go?

**Review by minkimcello — APPROVED** [review]

File: packages/fetch/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@effection/fetch",
-  "version": "2.0.0-preview.4",
+  "version": "1.0.0-preview.5",
```
Comment (jnicklas) [inline packages/fetch/package.json]: This is broken again

File: packages/atom/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@effection/atom",
-  "version": "2.0.0-preview.3",
+  "version": "2.0.0-preview.3-preview.4",
```
Comment (jnicklas) [inline packages/atom/package.json]: What happened here?

**Review by jnicklas — CHANGES_REQUESTED** [review]

File: packages/atom/package.json (hunk 2 above)
Comment (cowboyd) [reply packages/atom/package.json]: Whoa!

**Review by jnicklas — APPROVED** [review]

**Review by cowboyd — APPROVED** [review]

**Review by cowboyd — APPROVED** [review]

**Pull request merged**